src/commands/app/scaffold.ts

```diff
@@ -35,16 +37,54 @@ export default class AppScaffold extends Kommand {
       },
     ]);
 
-    await tasks.run();
-    this.log('')
-    this.logOk(`Scaffolding complete! Use ${chalk.grey(`cd ${destination} && npm run docker npm install`)} install dependencies and then ${chalk.grey(`npm run docker:dev`)} to run your application!`);
+    try {
+      await tasks.run();
+
+      this.log('')
+      this.logOk(`Scaffolding complete! Install dependencies with :
+        ${chalk.grey(`cd ${destination} && npm run docker npm install`)}
+        and run your application with:
+        ${chalk.grey('npm run docker:dev')}`)
+    }
+    catch (error: any) { }
   }
 
   async cloneTemplate(flavor: string, destination: string) {
-    await execute('git', 'clone', '--depth=1', 'https://github.com/kuzzleio/project-templates', '--branch', flavor, '--single-branch');
+    const templatesDir = '/tmp/kourou-template';
+    const assetName = `${flavor}.tar.gz`;
+    const link = `https://github.com/kuzzleio/project-templates/releases/latest/download/${assetName}`;
+
+    let directoryExist = true;
+    try {
+      await fsp.access(destination);
+    }
+    catch (error) {
+      directoryExist = false;
+    }
+
+    if (directoryExist) {
+      throw new Error(`Directory "${destination}" already exist`);
+    }
+
+    await fsp.mkdir(templatesDir, { recursive: true })
+
+    const response = await fetch(link);
+
+    if (! response.ok) {
+      throw new Error(`Scaffold for the flavor "${flavor}" does not exist`);
+    }
+
+    await fsp.writeFile(`${templatesDir}/${assetName}`, response.body as any);
```

Comment:
You should use `path.join` to join path so it will work on windows the day we want to port it